npm/extensions.bzl

```diff
@@ -30,15 +30,21 @@ def _npm_extension_impl(module_ctx):
     exclude_package_contents_config = _build_exclude_package_contents_config(module_ctx)
 
     for mod in module_ctx.modules:
+        replace_packages = {}
```

Comment:
Lets move this outside the loop like Alex mentioned in my PR: https://github.com/aspect-build/rules_js/pull/1886/files#r1715523135